lib/ash_admin/router.ex

```diff
@@ -59,25 +59,53 @@ defmodule AshAdmin.Router do
       end
   """
   defmacro ash_admin(path, opts \\ []) do
-    quote bind_quoted: [path: path, opts: opts] do
-      import Phoenix.LiveView.Router
-      live_socket_path = Keyword.get(opts, :live_socket_path, "/live")
-
-      live_session :ash_admin,
-        on_mount: List.wrap(opts[:on_mount]),
-        session:
-          {AshAdmin.Router, :__session__, [%{"prefix" => path}, List.wrap(opts[:session])]},
-        root_layout: {AshAdmin.Layouts, :root} do
-        live(
-          "#{path}/*route",
-          AshAdmin.PageLive,
-          :page,
-          private: %{live_socket_path: live_socket_path}
-        )
+    opts =
+      if Macro.quoted_literal?(opts) do
+        Macro.prewalk(opts, &expand_alias(&1, __CALLER__))
```

Comment:
There is `Macro.expand_literal` IIRC. Why are we doing this?

Reply:
I think we don't need this anymore, I was using the options to be able to forward the endpoint together, but this is no longer necessary, removed